pkg/arguments/arguments.go

```diff
@@ -314,19 +314,20 @@ func IsValidMode(modes []string, mode string) bool {
 	return false
 }
 
-func markGlobalFlagsHidden(command *cobra.Command, hidden ...string) {
+func deprecateRegion(command *cobra.Command, hidden ...string) {
```

Comment:
@hunterkepley The signature of this function feels a bit off. We're passing in a variadic list of fields to be hidden, but have a hard-coded deprecation message. I'd be tempted just to drop the `hidden` parameter and do a hardcoded lookup for the `region` field for now.